src/app/components/MainContent/index.jsx

```diff
@@ -41,7 +41,13 @@ const MainContent = ({ data }) => {
 MainContent.propTypes = {
   data: shape({
     model: shape({
-      blocks: arrayOf(any),
+      blocks: shape({
```

Comment:
Blocks is not an object. It is an array.

We should also only add `headline` and `text`, not `subheadling`, `image`, `video` - MainContent cannot render those.